src/tests/JIT/Directed/callconv/InstanceCallConvTest.cpp

```diff
@@ -38,33 +41,32 @@ class C
         height(height)
     {}
 
-    virtual SizeF GetSize()
+    virtual SizeF INSTANCE_CALLCONV GetSize()
```

Comment:
>virtual SizeF INSTANCE_CALLCONV GetSize() [](start = 4, length = 41)

Perhaps put some dummy arguments in these functions so we know the ret buffer is correctly placed?

Reply:
Will do.